### ISSUE TITLE
[3] LDP errors on bulkrax - staging

### ISSUE THREAD

**Issue by orangewolf** [scientist-softserv/louisville-hyku#77]
As of 2/3 - all csv imports fail on staging due to LDP errors

dev note: consider increasing pool timeout in fedora config ( == staging ops fedora.yml)

**Comment by orangewolf** [comment]
Shana January 2022
---
dev note: increase pool timeout in fedora config

**Comment by orangewolf** [comment]
Summer February 2022
---
How do we test this?

**Comment by orangewolf** [comment]
Summer February 2022
---
I just tried to create & import the Cardinal Text file as I was QAing #32

All the works failed with Ldp::NotFound errors. Is this the same as what this ticket is about? If so, I will push it back to ready for dev.I just tried to create & import the Cardinal Text file as I was QAing [#32 (closed)](https://github.com/notch8/louisville-hyku/-/issues/32)
All the works failed with Ldp::NotFound errors. Is this the same as what this ticket is about? If so, I will push it back to ready for dev.


![Image](https://user-images.githubusercontent.com/1054448/204005090-80d1c9b0-3d4e-401e-8634-1450cf92f2f9.png)

**Comment by orangewolf** [comment]
Shana February 2022
---
Notes from pairing with April:

This is not happening locally. This was not happening on staging yesterday.

Is this because the fedora path is hardcoding "single" in its path? Where and how does this happen?

Error from single.lv-hyku


![Image](https://user-images.githubusercontent.com/1054448/204005184-2cc44335-9fef-42cd-a7e9-e4412f03c6bd.png)

**Comment by orangewolf** [comment]
Shana February 2022
---
This issue has been resolved.

Summer,

For now, please retest using zipped csv as the files on the server is still WIP.

**Comment by orangewolf** [comment]
Rachel February 2022
---
As I recall, I provided the PDF in the file field for the Cardinal newspaper (and as the file), not the image files at the page level. So it may be a legitimate LDP error because there is no 19261001_2 etc. file - just a 19261001.pdf that covers all of the pages.

**Comment by orangewolf** [comment]
Alisha March 2022
---
@rachel.howard can you retest this?[@rachel.howard](https://github.com/rachel.howard) can you retest this?

**Comment by orangewolf** [comment]
Rachel March 2022
---
Is staging back up? I was waiting for the go-ahead from Shana.

**Comment by orangewolf** [comment]
Alisha May 2022
---
sorry that I never responded to this. yes, please test this again on staging.

**Comment by orangewolf** [comment]
Rachel May 2022
---
Retested several hours ago. No LDP errors but thumbnails/viewer images still aren't appearing.

**Comment by orangewolf** [comment]
Rachel May 2022
---
hmm, sounds like that may be a different problem. before creating a separate ticket to track that, I'd like to verify it's a new/different problem though.

is it a specific import and/or parent that isn't showing the child images? if the first child doesn't have an image attached I don't expect the parent to have a thumbnail either. can you try to replicate this on staging so that we have some code/logs/errors that we can look into?

**Comment by orangewolf** [comment]
Rachel May 2022
---
It's the Louisville Cardinal Newspapers collection on staging (https://lv-hyku-staging.notch8.cloud/collections/fbc441a3-ac7c-4efb-a92e-84ef69bf52da?locale=en). I realized today that the sample set had incomplete information in the parents field, which may have caused the issue yesterday. I reimported it today and it's still pending, possibly because it had to create JPEGs from the PDF and run the OCR etc.

**Comment by orangewolf** [comment]
Alisha May 2022
---
@rachel.howard can you send me that zip file that you used?[@rachel.howard](https://github.com/rachel.howard) can you send me that zip file that you used?

**Comment by orangewolf** [comment]
Rachel May 2022
---
Here it is! Since the links Randy set up to make accessible to your staging server have been deleted on your end, I have to test on staging with Zip files.

Rachel

**Comment by orangewolf** [comment]
Alisha May 2022
---
thanks! looks like this is a different issue altogether. it's unrelated ot the jdp errors that this ticket was created for.

after viewing the zip, I see there are 2 pdf's attached. sounds like the pdf isn't being converted into jpg derivatives anymore. (doesn't appear related to #109)

I created #116 to work and track this issue.thanks! looks like this is a different issue altogether. it's unrelated ot the jdp errors that this ticket was created for.
after viewing the zip, I see there are 2 pdf's attached. sounds like the pdf isn't being converted into jpg derivatives anymore. (doesn't appear related to [#109 (closed)](https://github.com/notch8/louisville-hyku/-/issues/109))
I created [#116 (closed)](https://github.com/notch8/louisville-hyku/-/issues/116) to work and track this issue.
